spec/lib/services/obm-service-spec.js

```diff
@@ -89,4 +90,147 @@ describe('OBM Service', function() {
             });
         });
     });
+
+    describe('Check invalid obm service', function() {
+        var waterline;
+
+        before('check invalid obm service before', function() {
+            waterline = helper.injector.get('Services.Waterline');
+            waterline.skus.findOne = sinon.stub();
```

Comment:
Does this not fail without doing `sinon.stub().resolves` ? I would expect calling `.then` on it to not work otherwise.